core/storage/suggestion/gae_models_test.py

```diff
@@ -6028,7 +6028,18 @@ class SortChoicesUnitTest(test_utils.GenericTestBase):
     """
 
     def test_enum_dictionary_match(self) -> None:
+        enum_values = []
+        dict_values = []
         for enum_member in suggestion_models.SortChoices:
+            enum_values.append(enum_member.value)
             self.assertTrue(
                 enum_member.value in constants.CD_ADMIN_STATS_SORT_OPTIONS
             )
+
+        for key in constants.CD_ADMIN_STATS_SORT_OPTIONS:
```

Comment:
Just do dict_values = constants.CD_ADMIN_STATS_SORT_OPTIONS.keys()

Reply:
If we do this, we can't use the sort() function. I think we could keep the existing method as well.

Reply:
I don't understand, why can't you use the sort() function? Don't you end up with a list of strings either way (perhaps after casting with list())?

Reply:
Casting it into a list works. Thanks.